docs/components/movement-sensor/adxl345.md

```diff
@@ -143,13 +142,13 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                  | Type   | Inclusion    | Description |
-| --------------------- | ------ | ------------ | ----------- |
-| `board`               | string | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
-| `i2c_bus`             | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
-| `use_alternate_i2c_address` | bool   | Optional     | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [Freefall attributes](#freefall-attributes). |
+| Name | Type   | Inclusion    | Description |
+| ---- | ------ | ------------ | ----------- |
+| `i2c_bus` | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
```

Comment:
The point of the RDK change is that you don't configure the I2C bus on the board component any more, and this component is no longer dependent on the board at all. I like the phrasing in https://docs.viam.com/components/board/pca9685/: "The index of the I2C bus on the board your device is connected to. Often a number. Example: `"1"`"

Reply:
Oh yes, thank you for catching this! Updating